### PR TITLE
Update global setting range values

### PR DESCRIPTION
**Pull request by KaichenWang**
**Why are these changes introduced?**

Update setting ranges for:
- Buttons
- Inputs
- Layout
- Media
- Content containers

**What approach did you take?**

Use updated values and naming

**Other considerations**

May change in the future

**Demo links**

- [Store](https://os2-demo.myshopify.com/?preview_theme_id=126958305302)
- [Editor](https://os2-demo.myshopify.com/admin/themes/126958305302/editor)

**Checklist**
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

File: config/settings_schema.json
Comment (tyleralsbury) [inline config/settings_schema.json:484]: I'm wondering if the steps here are too large for blur radius.  I feel like each `px` you add to a blur radius adds a lot.  Not a strongly held opinion, so I defer to @melissaperreault and @KaichenWang on the decision.

File: config/settings_schema.json
Comment (tyleralsbury) [inline config/settings_schema.json:817]: I like the larger max, I think it will lead to some nice open layouts.

File: config/settings_schema.json
Comment (tyleralsbury) [inline config/settings_schema.json:953]: I wonder if 100 is too large of a blur radius.  Maybe not, but it is massive.  I'm okay with it, but I am curious if folks will actually use it.

File: layout/password.liquid
Comment (tyleralsbury) [inline layout/password.liquid:77]: I like the idea of removing buttons / input padding - it will ensure accessible buttons and inputs.  I'm trying to think if there would be a valid art direction for very narrow buttons or inputs, but I think an opinion on function over form makes sense for controls like this.

**Review by tyleralsbury — APPROVED** [review]
A couple of comments.  Nothing blocking merge as far as I can see.

File: config/settings_schema.json
Comment (melissaperreault) [inline config/settings_schema.json:496]:
```suggestion
        "content": "Border"
```
We have the singular form, like for `Shadow`.

File: config/settings_schema.json
Comment (melissaperreault) [inline config/settings_schema.json:330]:
```suggestion
        "content": "Border"
```

File: config/settings_schema.json
Comment (melissaperreault) [inline config/settings_schema.json:413]:
```suggestion
        "content": "Border"
```

**Review by melissaperreault — COMMENTED** [review]
Thank you Kai! 🎉 

Looking good! I added a few notes feel free to disregard if you feel they should be tackled from another PR. 🙏 

**Buttons, Inputs, Content containers, and Media**
- [x] Have the `Thickness` setting appear before `Opacity`


**Layout**
- [ ] Have the `horizontal` setting display before `vertical` for both Desktop and Mobile grid
- [ ] Let's adjust the Mobile grid values for both axis to match the system grid 

> 

    max": 40,
    "step": 4,
    "default": 4,

File: config/settings_schema.json
Comment (melissaperreault) [inline config/settings_schema.json:400]:
```suggestion
        "max": 20,
```
For the same reasons of Inputs

File: layout/password.liquid
Comment (melissaperreault) [reply layout/password.liquid:77]: We are confident that the initial button spacing was scalable for most brands. We are insuring to let merchants have a well balanced label over the button background to grab buyers attention. We'll monitor though if that becomes a common request.

There are some tweaks to make to buttons though with the flexibility settings, coming in another issue!

**Comment by KaichenWang** [comment]
Since this PR only changes setting range values and copy, I'm going to merge. I believe it's best to continue adjusting and discussing in a follow up.

**Pull request merged**
